services/s3/pom.xml

```diff
@@ -93,5 +93,15 @@
             <artifactId>assertj-core</artifactId>
             <version>${assertj.version}</version>
         </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
```

Comment:
Why do we need this dependency?

Reply:
I needed it to debug S3 tests that were failing, and I figured it would be a good thing to leave in. There was already a log4j.properties, so I figured logging was meant to be enabled. Let me know if you want me to remove it.

Reply:
Don't you only need the slf4j one?

Reply:
Let me triple-check, I thought the slf4j-log4j12 was just the bridge.